deployment/helm/ditto/templates/connectivity-configmap.yaml

```diff
@@ -17,6 +17,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ $releaseName }}-service-config-connectivity-extension-conf
+  namespace: {{ $root.Release.Namespace }}
```

Comment:
what is the difference between configmaps and other resources that requires the use of $root?
I guess this question is valid for all occurences :)

Reply:
I just used everywhere where a variable `$root` was defined this variable instead of `.`

It is only necessary for places where we e.g. iterate over something - in that case `.` is no longer the "root", but is scoped to the current element being iterated over.